tests/configs/test-config.yaml

```diff
@@ -0,0 +1,60 @@
+# SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: Unlicense
+
+# Parameters of repository traversal.
+traversal:
+  # Files and folders which we pretend do not exist
+  # (so they are neither analyzed nor can be referenced).
+  ignored:
+    # Git files
+    - .git
+
+    # Stack files
+    - .stack-work
+
+    # Test files
+    - tests/markdowns/with-annotations
+
+# Verification parameters.
+verification:
+  # On 'anchor not found' error, how much similar anchors should be displayed as
+  # hint. Number should be between 0 and 1, larger value means stricter filter.
+  anchorSimilarityThreshold: 0.5
+
+  # When checking external references, how long to wait on request before
+  # declaring "Response timeout".
+  externalRefCheckTimeout: 10s
+
+  # Prefixes of files, references in which should not be analyzed.
+  notScanned:
+    # GitHub-specific files
+    - .github/pull_request_template.md
+    - .github/issue_template.md
+    - .github/PULL_REQUEST_TEMPLATE
+    - .github/ISSUE_TEMPLATE
+
+    # GitLab-specific files
+    - .gitlab/merge_request_templates/
+    - .gitlab/issue_templates/
+
+  # Glob patterns describing the files which do not physically exist in the
+  # repository but should be treated as existing nevertheless.
+  virtualFiles:
+    # GitHub pages
+    - ../../../issues
+    - ../../../issues/*
+    - ../../../pulls
+    - ../../../pulls/*
+
+    # GitLab pages
+    - ../../issues
+    - ../../issues/*
+    - ../../merge_requests
+    - ../../merge_requests/*
+
+  # Regular expressions that match external references that have to be ignored.
+  # It is an optional parameter, so it can be omitted.
+  ignoreRefs:
+    - https://bad.((external.)?)reference(/?)
+    - https://bad.reference.(ru|com)
```

Comment:
It is better to avoid copy-pasting the default config and doing small changes - just for the sake of tests, because when we later add more test configs and more options, maintaining all the config files would become extremely tiresome.

I see 3 options for how to improve this:
1. Remove comments and all the unnecessary entries and hope that one day we make all the fields here optional.
2. Initialize the necessary part of config in Haskell, e.g. `config = defConfig{ vcIgnoreRefs = Aeson.decode "<config text>" }`. You may need to add lenses for config to make it easier to update fields.
3. An ideal option - like 2, but do `config = defConfig{ vsIgnoreRefs = ["regex1", "regex2"] }`, and add tests on JSON decoding of regex separately.